### DISCUSSION TITLE
不支持 mysql自带的系统表

### DISCUSSION THREAD

**Discussion by mawenbin666** [actiontech/dble#2111] [Ideas]
* **dble version：2.19.07.0

开发经常要用到mysql自带的系统表，比如，mysql，information_shcema 等几个库里面的表，期待下个最新版本可以支持下，谢谢。

**Comment by yanhuqing666** [comment]
There are too many tables in schema  `mysql`，`information_shcema`, and some information should be generated by dble.
So it will cost too much time but most of them are useless.
Please tell us the most important tables of these you think, not the entire schema.
